File: tests/fixtures/egress/s.go
package main
import "net/http"
func main() {
	resp, _ := http.Get("https://api.example.com/v1/x")
	req, _ := http.NewRequest("GET", url, nil)
	res, _ := client.Do(req)
	http.Post(url, "application/json", body)
	_ = resp; _ = res
}

File: tests/fixtures/egress/s.py
import requests, httpx, urllib.request
r = requests.get("https://api.example.com/v1/x")
p = requests.post(url, json=body)
h = httpx.get("https://api.example.com/v1/y")
u = urllib.request.urlopen("https://api.example.com/v1/z")
s = requests.Session()
